website/www/site/content/en/contribute/_index.md

```diff
@@ -21,55 +24,61 @@ See the License for the specific language governing permissions and
+...and many more. Most importantly, if you have an idea of how to contribute, then do it!
 
- - ask or answer questions on [user@beam.apache.org](/community/contact-us/) or
-[stackoverflow](https://stackoverflow.com/questions/tagged/apache-beam)
- - review proposed design ideas on [dev@beam.apache.org](/community/contact-us/)
- - improve the documentation
- - file [bug reports](https://issues.apache.org/jira/projects/BEAM/issues)
- - test releases
- - review [changes](https://github.com/apache/beam/pulls)
- - write new examples
- - improve your favorite language SDK (Java, Python, Go, etc)
- - improve specific runners (Apache Flink, Apache Spark, Google
-   Cloud Dataflow, etc)
- - improve or add IO connectors
- - add new transform libraries (statistics, ML, image processing, etc)
- - work on the core programming model (what is a Beam pipeline and how does it
-   run?)
- - improve the developer experience (for example, Windows guides)
- - add answers to the [contribution FAQ](
- https://cwiki.apache.org/confluence/display/BEAM/Contributor+FAQ)
- - organize local meetups of users or contributors to Apache Beam
+</div>
 
-Most importantly, if you have an idea of how to contribute, then do it!
+<a class="arrow-list-header" data-toggle="collapse" href="#collapseContributing" role="button" aria-expanded="false" aria-controls="collapseContributing">
+   {{< figure src="/images/arrow-icon_list.svg">}}
 
 ## Contributing code
 
+  </a>
+
+<div class="collapse dont-collapse-sm" id="collapseContributing">
+
 Below is a tutorial for contributing code to Beam, covering our tools and typical process in
 detail.
 
 ### Prerequisites
 
-To contribute code, you need
+* a GitHub account
+* a Linux, macOS, or Microsoft Windows development environment with Java JDK 8 installed
+* [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
+  changes locally
+* [Go](https://golang.org) 1.12 or later installed for Go SDK development
+* Python 3.6, 3.7, and 3.8. Yes, you need all four versions installed.
+
+  pip, setuptools, virtualenv, and tox installed for Python development
+
+* for large contributions, a signed [Individual Contributor License
+  Agreement](https://www.apache.org/licenses/icla.pdf) (ICLA) to the Apache
+  Software Foundation (ASF).
 
- - a GitHub account
- - a Linux, macOS, or Microsoft Windows development environment with Java JDK 8 installed
- - [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
-   changes locally
- - [Go](https://golang.org) 1.12 or later installed for Go SDK development
- - Python 3.6, 3.7, and 3.8. Yes, you need all three versions installed.
-    - pip, setuptools, virtualenv, and tox installed for Python development
- - for large contributions, a signed [Individual Contributor License
-   Agreement](https://www.apache.org/licenses/icla.pdf) (ICLA) to the Apache
-   Software Foundation (ASF).
```

Comment:
Can we just keep this section the same as it was before? The change doesn't seem to be substantive, just changing bullet points.

Reply:
Okay I will adjust the layout to content from website-revamp branch and the community will consider changing it later

Reply:
Thank you